src/action.js

```diff
@@ -65,46 +57,19 @@ module.exports = async function run() {
       return logInfo(`${pkgName} is excluded, skipping.`)
     }
 
-    if (API_URL !== DEFAULT_API_URL &&
-      pkgName === 'github-action-merge-dependabot' &&
-      isMajorRelease(pr)) {
+    if (pkgName === 'github-action-merge-dependabot' && isMajorRelease(pr)) {
```

Comment:
shall we not move this check above, right after the check on whether it's a dependabot PR?

Reply:
I added this check here after the exclude packages check because it seems less confusing for those users that have added the `github-action-merge-dependabot` into the `exclude` option.

Reply:
don't we want to fail independently of whether the package would eventually be automerged or not? ok maybe it doesn't make a big difference